### PR TITLE
Cosmosdb pitr timestamp fixes

### PR DESCRIPTION
**Pull request by kavskalyan**
---

1. Making utc as the default timezone when the timezone is not provided for the restore timestamps
2. Added the `--name` parameter to `az cosmosdb restorable-database-account list` to return all the restorable database accounts with the given name
3. Updating the sdk to the released version.

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Cosmosdb

**Review by MehaKaushik — APPROVED** [review]

**Comment by kavskalyan** [comment]
@evelyn-ys Can you please review this PR. Thanks :)

File: src/cosmosdb-preview/azext_cosmosdb_preview/actions.py [hunk 1]
```diff
@@ -91,3 +91,33 @@ def __call__(self, parser, namespace, values, option_string=None):
...
+        import dateutil.tz
+
+        accepted_formats = []
+        accepted_formats.append('date (yyyy-mm-dd)')
+        accepted_formats.append('time (hh:mm:ss.xxxxx)')
+        accepted_formats.append('timezone (+/-hh:mm)')
+        help_string = 'Format: ' + ' '.join(accepted_formats)
+        value_string = ''.join(values)
+        dt_val = None
+        print(values)
```
Comment (evelyn-ys) [inline src/cosmosdb-preview/azext_cosmosdb_preview/actions.py:109]: Is this `print` for debug?

**Review by evelyn-ys — CHANGES_REQUESTED** [review]
Have you updated `setup.py` to upgrade version?

File: src/cosmosdb-preview/azext_cosmosdb_preview/actions.py (hunk 1 above)
Comment (kavskalyan) [reply src/cosmosdb-preview/azext_cosmosdb_preview/actions.py]: Thanks for catching. I removed it now.

**Comment by kavskalyan** [comment]
> Have you updated `setup.py` to upgrade version?

Missed it. Upgraded it now.

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**